APMrover2/Parameters.h

```diff
@@ -370,6 +370,9 @@ class ParametersG2 {
     // mission behave
     AP_Int8 mis_done_behave;
 
+    // Hold FS behave
+    AP_Int8 r_options;
```

Comment:
This is a really small storage space for options, only gives you 7 options (easily) total. It's probably worth making this a Int32.

Reply:
done in new commit